docs/content/s3.md

```diff
@@ -380,7 +380,14 @@ However for objects which were uploaded as multipart uploads or with
 server side encryption (SSE-AWS or SSE-C) the `ETag` header is no
 longer the MD5 sum of the data, so rclone adds an additional piece of
 metadata `X-Amz-Meta-Md5chksum` which is a base64 encoded MD5 hash (in
-the same format as is required for `Content-MD5`).
+the same format as is required for `Content-MD5`).  You can use base64 -d and hexdump to check this value manually:
+
+    echo 'VWTGdNx3LyXQDfA0e2Edxw==' | base64 -d | hexdump
+
+or use native commands to verify:
```

Comment:
I think the second part of this doesn't belong here in this discussion of hashes so can you remove `or use native commands to verify:` to the end of the patch?

Or maybe condense to

> You can use `rclone check` to verify the hashes are OK.

Reply:
done.  let me know if this works.